components/Publication/PublicationForm.js

```diff
@@ -345,13 +201,14 @@ const Form = ({
               {links.map((link, i) => (
                 <li key={i}>
                   <Interaction.P
-                    style={{
-                      color: link.errors.length
-                        ? colors.error
+                    {...colorScheme.set(
+                      'color',
+                      link.errors.length
+                        ? 'error'
                         : link.warnings.length
-                        ? colors.social
+                        ? '#e0aa14'
```

Comment:
Where does this magic color come from?

Reply:
colors are so messed up :D, because there's no warning color, we use colors.social for warnings, but I wanted a colored box with white text on top and color.social looks bad that way, so I invented a new one 👯 . Clean approach would be to introduce a "warning" color in styleguide, but I was lazy, because I didn't want to update the styleguide for this. Do you think we should?

Reply:
well, we should... but I see your point. Maybe let's create a small ticket for this in trello and tackle it bei gelegenheit?

Reply:
done